frontend/src/pages/org/archived-item-detail/ui/qa.ts

```diff
@@ -1,216 +1,718 @@
+                        <btrix-menu-item-link href="#" download>
+                          <sl-icon name="download" slot="prefix"></sl-icon>
+                          ${msg("Download QA Run")}
+                        </btrix-menu-item-link>
+                        <sl-divider></sl-divider>
+                      `}
+                  <sl-menu-item
+                    @click=${() => {
+                      this.deleting = run.id;
+                      void this.deleteQADialog?.show();
+                    }}
+                    style="--sl-color-neutral-700: var(--danger)"
+                  >
+                    <sl-icon name="trash3" slot="prefix"></sl-icon>
+                    ${msg("Delete Item")}
+                  </sl-menu-item>
+                </sl-menu>
+              </btrix-overflow-dropdown>
+            </div>
+          </btrix-table-cell>
+        </btrix-table-row>
+      `,
+    );
+  };
+
+  private readonly renderDeleteConfirmDialog = () => {
+    const runToBeDeleted = this.qaRuns?.find((run) => run.id === this.deleting);
+
+    if (!runToBeDeleted) return;
+
+    return html`
+      <btrix-dialog id="deleteQARunDialog" .label=${msg("Delete QA Run?")} open>
+        <b class="font-semibold"
+          >${msg("All of the data included in this QA run will be deleted.")}</b
+        >
+        <div>
+          ${msg(
+            str`This QA run includes data for ${runToBeDeleted.stats.done} ${pluralize(runToBeDeleted.stats.done, { zero: msg("pages", { desc: 'plural form of "page" for zero pages', id: "pages.plural.zero" }), one: msg("page"), two: msg("pages", { desc: 'plural form of "page" for two pages', id: "pages.plural.two" }), few: msg("pages", { desc: 'plural form of "page" for few pages', id: "pages.plural.few" }), many: msg("pages", { desc: 'plural form of "page" for many pages', id: "pages.plural.many" }), other: msg("pages", { desc: 'plural form of "page" for multiple/other pages', id: "pages.plural.other" }) })} and was started on `,
           )}
-        </btrix-desc-list-item>
-      </btrix-desc-list>
-    </div>
-    <section class="mb-7">
-      <div class="flex items-center gap-1">
-        <h4 class="text-base font-semibold leading-8">${msg("Analysis")}</h4>
-        <btrix-qa-run-dropdown
-          .items=${finishedQARuns}
-          selectedId=${qaRunId || ""}
-          @btrix-select=${(e: CustomEvent<SelectDetail>) =>
-            (qaRunId = e.detail.item.id)}
-        ></btrix-qa-run-dropdown>
+          <sl-format-date
+            lang=${getLocale()}
+            date=${`${runToBeDeleted.started}Z`}
+            month="2-digit"
+            day="2-digit"
+            year="2-digit"
+            hour="2-digit"
+            minute="2-digit"
+          ></sl-format-date>
+          ${msg("by")} ${runToBeDeleted.userName}.
+        </div>
+        <div slot="footer" class="flex justify-between">
+          <sl-button
+            size="small"
+            variant="primary"
```

Comment:
Delete confirmation button variants are a bit all over the place in the app, this might be a good place to start standardizing-- @Shrinks99 what should the button variants be for "Cancel" and "Delete" in the confirmation dialog?

Reply:
Cancel and Delete are both destructive actions.  We've been using red for those, would use the danger variant for both?

Reply:
Sure, makes sense!

Reply:
Addressed in 2bc2e0e